script/version.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 prefix=`cat VERSION` # Retrieve the version prefix
-timestamp=`date +%s` # Calculate the current timestamp
+timestamp=`date +%Y%m%d%TH%M%S` # Calculate the current timestamp in ISO8601 format
```

Comment:
Can we use a flat string of numbers for compatibility simplicity? Also I think there's an issue with the date format, and we should standardize to UTC:

```suggestion
timestamp=`date -u +%Y%m%d%H%M%S` # Calculate the current UTC timestamp
```

The ISO8601 format isn't compatible with `Gem::Version` 😞 

```
13:30:37 ~/dev/connectors> date +%Y%m%d%TH%M%S
2022041214:07:12H0712
14:07:12 ~/dev/connectors>
```

```
irb(main):007:0> Gem::Version.new('8.2.0.0-2022041214:07:12H0712')
Traceback (most recent call last):
       14: from /Users/rossbell/dev/ent-search/bin/enterprise-search-internal:15:in `<main>'
       13: from /Users/rossbell/dev/ent-search/shared_togo/lib/shared_togo/cli.rb:112:in `run!'
       12: from /Users/rossbell/dev/ent-search/shared_togo/lib/shared_togo/cli.rb:125:in `run_command'
       11: from /Users/rossbell/dev/ent-search/shared_togo/lib/shared_togo/cli.rb:143:in `run_supported_command'
       10: from /Users/rossbell/dev/ent-search/shared_togo/lib/shared_togo/cli/command.rb:10:in `run_and_exit'
        9: from /Users/rossbell/dev/ent-search/shared_togo/lib/shared_togo/cli/rails_console_command.rb:22:in `run'
        8: from org/jruby/RubyKernel.java:1237:in `catch'
        7: from org/jruby/RubyKernel.java:1237:in `catch'
        6: from org/jruby/RubyKernel.java:1507:in `loop'
        5: from org/jruby/RubyKernel.java:1091:in `eval'
        4: from (irb):7:in `evaluate'
        3: from /Users/rossbell/.rbenv/versions/jruby-9.3.3.0/lib/ruby/stdlib/rubygems/version.rb:205:in `new'
        2: from org/jruby/RubyClass.java:907:in `new'
        1: from /Users/rossbell/.rbenv/versions/jruby-9.3.3.0/lib/ruby/stdlib/rubygems/version.rb:214:in `initialize'
ArgumentError (Malformed version number string 8.2.0.0-2022041214:07:12H0712)
irb(main):008:0>
```

However, `20220412140712` would work with _anything_ and is easily human readable.

Reply:
Ah shit. I put the `T` in the wrong place for my suggestion.
```
Seans-MacBook-Pro-2:ent-search seanstory$ date +%Y%m%dT%H%M%S
20220412T142016
Seans-MacBook-Pro-2:ent-search seanstory$ date +%Y%m%d%TH%M%S
2022041214:21:08H2108
Seans-MacBook-Pro-2:ent-search seanstory$ 
```

The former is what I would like, I think. (Also with the `-u`, good call)

Reply:
Got it, it still works with `T` and `Z`. I'll update my recommended change.